src/ui/manager.tsx

```diff
@@ -47,7 +47,6 @@ const WebReaderContent: React.FC<ReaderReturn & ReaderManagerArguments> = ({
         />
       </PageButton>
       <Flex
-        width="100vw"
```

Comment:
This was removed because of a CSS issue that showed up in SFR.  I'm surprised that it didn't show up anywhere else - I did a cursory check to make sure it wasn't breaking anything in the example app, but please double check

Reply:
Yeah, it seems fine without the extra CSS.